tests/observability/metrics/test_vms_metrics.py

```diff
@@ -37,15 +37,15 @@
     USED,
 )
 from utilities.infra import get_node_selector_dict
+from utilities.monitoring import get_metrics_value
 from utilities.virt import VirtualMachineForTests, fedora_vm_body, running_vm
 
 LOGGER = logging.getLogger(__name__)
 
 
 def get_vm_metric_value(prometheus, metric, vm):
-    query = f"{metric}{{name='{vm.name}'}}"
-    metric_results = prometheus.query(query=query)["data"]["result"]
-    return int(metric_results[0]["value"][1]) if metric_results else 0
+    promql = f"{metric}{{name='{vm.name}'}}"
```

Comment:
no need to save, please use directly in the line below.

Reply:
Thanks for review @dbasunag . Fixed.